google/resource_monitoring_uptime_check_config.go

```diff
@@ -218,6 +218,10 @@ func resourceMonitoringUptimeCheckConfig() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"update_mask": {
```

Comment:
@rambleraptor I don't think this actually does what update_mask is looking for. Update mask usually needs to be passed as part of the url params, not in the body of the request. Also for TF it shouldn't be a user defined field but instead be generated based on what fields have changed.

Making it a user defined field will be bad in terraform because terraform will try to persist it to state and maintain a diff of when it gets changed. This would result in TF submitting an update to an api if the update_mask changed, which I isn't desired.